README.md

```diff
@@ -95,8 +95,8 @@ stripe:
   private_key: sk_test_YOUR_TEST_STRIPE_KEY
   signing_secret: whsec_YOUR_SIGNING_SECRET
   price_ids:
-    part_time_plan: price_YOUR_PRODUCT_PRICE_ID
-    full_time_plan: price_ANOTHER_PRODUCT_PRICE_ID
+    part_time: price_YOUR_PRODUCT_PRICE_ID
+    full_time: price_ANOTHER_PRODUCT_PRICE_ID
```

Comment:
What's the reasoning behind this change?

Reply:
We talked about this in person at the hackathon. When I tried using `part_time_plan` and `full_time_plan` under the stripe price_ids section of the development credentials file I was getting an error when trying to check out. When we switched it to `part_time` and `full_time` it worked and you remembered this had changed at some point and asked me to update the readme.

Reply:
Beautiful! I totally forgot about our conversation at the hackathon. Thank you for remembering it. :D

Reply:
👍